src/sciwrappers.c

```diff
@@ -954,6 +958,17 @@ void sci_set_font(ScintillaObject *sci, gint style, const gchar *font, gint size
 }
 
 
+/** Gets the font size for a particular style.
+ * @param sci Scintilla widget.
+ * @param style The style.
+ * @return The font size. */
+GEANY_API_SYMBOL
+gint sci_get_fontsize(ScintillaObject *sci, gint style)
```

Comment:
`sci_get_font_size()`.

Also, why is it part of the plugin API?  Not that I don't want it, but we prefer to have a good reason for adding stuff to the API so we don't have to maintain something useless.

Reply:
Done (long time ago), no longer part of the API.